Source/SwiftLintFramework/Rules/OperatorUsageWhitespaceRule.swift

```diff
@@ -105,7 +107,7 @@ public struct OperatorUsageWhitespaceRule: OptInRule, CorrectableRule, Configura
             zeroSpaces + trailingVariableOrNumber
         let excludingPattern = "(?:\(genericPattern)|\(validRangePattern))"
 
-        let kinds = SyntaxKind.commentAndStringKinds()
+        let kinds = SyntaxKind.commentAndStringKinds() + [.objectLiteral]
```

Comment:
yes, basically "non-code kinds", and check to see if there are even more kinds that fall into that category.